src/ui/configs/index.ts

```diff
@@ -2,4 +2,5 @@ import defaultIcons from 'ui/configs/common/icons';
 
 export default {
     icons: defaultIcons,
+    isZitadelEnabled: false,
```

Comment:
It will be better to use more common name then `Zitadel`. Tomorrow we we may want to add different identity platform.

Reply:
I thought about it, let's leave as is for now as it will require to rename lots of places. I think if we decide to make IAM's pluggable we easily refactor names and make them more common